### ISSUE TITLE
Parthenocissus vitacea

### ISSUE THREAD

**Issue by MichaelOldham** [Canadensys/vascan-data#2360]
Pringle (2010) suggests that P. inserta is the correct name.  Is it possible to attach items?  I will send article separately.

http://data.canadensys.net/vascan/taxon/21584?lang=en

MIKE OLDHAM

**Comment by brouille** [comment]
You are one step behind. The FNA draft clearly discussed why the usage of inserta was erroneous. The taxonomy in VASCAN is up-to-date as of 2015.

There is apparently a difficulty in attaching docs with Github. Please do not send this one. I already saw it� and dealt with it.

Luc

De : MichaelOldham <notifications@github.com<mailto:notifications@github.com>>
R�pondre � : Canadensys/vascan-data <reply@reply.github.com<mailto:reply@reply.github.com>>
Date : mardi 14 avril 2015 14:23
� : Canadensys/vascan-data <vascan-data@noreply.github.com<mailto:vascan-data@noreply.github.com>>
Objet : [vascan-data] Parthenocissus vitacea http://data.canadensys.net/vascan/taxon/21584?lang=en (#2360)


Pringle (2010) suggests that P. inserta is the correct name. Is it possible to attach items? I will send article separately.

MIKE OLDHAM

�

**Comment by MichaelOldham** [comment]
Sorry Luc,



Didn’t check the FNA draft!!



mike

**Comment by MichaelOldham** [comment]
Interesting that the FNA draft does not cite the Pringle (2010) paper, though perhaps this has been added



mike

**Comment by brouille** [comment]
Mike:

We did not asked this paper to be cited, according to my copy of the ECAN edits. The comment indeed does not take the Pringle comment into account.

I will send this to Geoff Levin, taxon editor for this group. This could change the taxonomy used… or not.

Luc

De : MichaelOldham <notifications@github.com<mailto:notifications@github.com>>
Répondre à : Canadensys/vascan-data <reply@reply.github.com<mailto:reply@reply.github.com>>
Date : mardi 14 avril 2015 14:29
À : Canadensys/vascan-data <vascan-data@noreply.github.com<mailto:vascan-data@noreply.github.com>>
Cc : Luc Brouillet <luc.brouillet@umontreal.ca<mailto:luc.brouillet@umontreal.ca>>
Objet : Re: [vascan-data] Parthenocissus vitacea (#2360)

Interesting that the FNA draft does not cite the Pringle (2010) paper, though perhaps this has been added



mike
